runner/tests/operator/copy_operator/test_copy_operator.py

```diff
@@ -0,0 +1,197 @@
+        run_input = operator.create_input()
+        expected_input = {
+            'class': 'File',
+            'path': self.file1.path
+            }
+        self.assertDictEqual(run_input, expected_input)
+
+    # disable job submission to Ridgeback
+    @patch('runner.tasks.submit_job')
+    def test_create_copy_operator_run(self, submit_job):
+        """
+        Test case for creating a Copy Operator run
+        """
+        # there should be no Runs
+        number_of_runs = len(Run.objects.all())
+        self.assertEqual(number_of_runs, 0)
+
+        # make a fake http request with some data
+        # and send it to the API endpoint for starting a run
+        request = MockRequest()
+        request.data = {
+            'request_ids': ['1'],
+            'pipeline_name': CopyOperator._pipeline_name
+            }
+        view = OperatorViewSet()
+        response = view.post(request)
+
+        # there should be 1 run now
+        number_of_runs = len(Run.objects.all())
+        self.assertEqual(number_of_runs, 1)
+
+
+    def test_live_copy_operator_run(self):
+        """
+        A test case for submitting a job to Ridgeback for a custom file that is passed from the environment
+
+        $ COPY_FILE=1 python manage.py test runner.tests.operator.copy_operator.test_copy_operator.TestCopyOperator.test_live_copy_operator_run
+        """
+        enable_testcase = os.environ.get('COPY_FILE')
+        request_id = 'test_live_copy_operator_run'
+
+        if enable_testcase:
+            print(">>> running TestCopyOperator.test_live_copy_operator_run")
+
+            # make a file
+            path = os.path.join(self.tmpdir, "file.txt")
+            with open(path, 'w') as f:
+                f.write('some data\n')
+
+            # make db entries
+            file_instance = File.objects.create(
+                file_name = os.path.basename(path),
+                path = path,
+                file_group = self.filegroup
+            )
+            filemetadata_instance = FileMetadata.objects.create(
+                file = file_instance,
+                metadata = {'requestId' : request_id}
+            )
+
+            # make a request to start the run
+            request = MockRequest()
+            request.data = {
+                'request_ids': [request_id],
+                'pipeline_name': CopyOperator._pipeline_name
+                }
+            view = OperatorViewSet()
+            response = view.post(request)
+            print(">>> response: ", response, response.data)
+
+            self.assertEqual(response.status_code, 200)
+
+            # there should be 1 run now
+            self.assertEqual(len(Run.objects.all()), 1)
+            run_instance = Run.objects.all().first()
+
+            count = 0
+            while True:
+                count += 1
+                print('>>> checking job status ({})'.format(count))
+                check_jobs_status()
```

Comment:
@sivkovic how do you make the Run get submitted to Ridgeback here? I keep getting 

```
ERROR:runner.tasks:Job  not submitted
```